.circleci/config.yml

```diff
@@ -136,10 +136,11 @@ jobs:
             gotestsum --junitfile "$TEST_RESULTS/gotestsum-report.xml" -- ./... -p 1 -timeout 20m -failfast \
               -enable-enterprise \
               -enable-multi-cluster \
+              -run TestControllerNamespaces\
               -kubeconfig="$primary_kubeconfig" \
               -secondary-kubeconfig="$secondary_kubeconfig" \
               -debug-directory="$TEST_RESULTS/debug" \
-              -consul-k8s-image=hashicorpdev/consul-k8s:latest
+              -consul-k8s-image=lkysow/consul-k8s-dev:sep09-crd-ent6
```

Comment:
This should change to `hashicorpdev/consul-k8s:crd-controller-base-latest` before merge